src/context/reducers/noteReducer.js

```diff
@@ -10,13 +10,21 @@ export const NoteReducer = (noteState, action) => {
             return { ...noteState, label: action.payload }
```

Comment:
just a suggestion , destructure action into {type , payload } instead writing action. in every case

Reply:
Okay